### PR TITLE
Switch images based on theme

### PR DESCRIPTION
**Pull request by Blind4Basics**
Testing for #71  (~~waiting for the rendered preview version to see if that works~~... it doesn't x) )

### CODE REVIEW

**Comment by Blind4Basics** [comment]
'doesn't seem to work... :/

image

What did I mess?

**Comment by Blind4Basics** [comment]
...tried adding the `./img` folder at root level: doesn't work either... (desperate fail, desperate attempt... XD )

(note: some files to remove from that branch, when it's done)

**Comment by Blind4Basics** [comment]
looks like it's ok, now :)
I removed all the pending files/folders. Waiting for the checks, now...

File: content/languages/python/codewars-test/index.md [hunk 1]
```diff
@@ -46,7 +46,16 @@ def rnd_tests():
...
 
-![Python Test Framework Example](./python-test-framework-example.png)
+<div class="block dark:hidden>
+
+![Output window example](./python-test-framework-example-light.png)
+
+</div>
+<div class="hidden dark:block">
+
+![Output window example](./python-test-framework-example-light.png)
```
Comment (kazk) [inline content/languages/python/codewars-test/index.md:56]:
```suggestion
![Output window example](./python-test-framework-example-dark.png)
```

Comment (Blind4Basics) [reply content/languages/python/codewars-test/index.md]: woops... x)

File: content/languages/python/codewars-test/index.md [hunk 2]
```diff
@@ -46,7 +46,16 @@ def rnd_tests():
 
 The above produces an output similar to the following:
 
-![Python Test Framework Example](./python-test-framework-example.png)
+<div class="block dark:hidden>
```
Comment (kazk) [inline content/languages/python/codewars-test/index.md:49]:
```suggestion
<div class="block dark:hidden">
```

**Pull request merged**